setup.cfg

```diff
@@ -31,13 +31,12 @@ python_requires = >=3.9
 cmdclass =
   install = install.AWEInstall
 install_requires =
-  awe_lexica
+  awe_lexica @ git+https://github.com/ArgLab/AWE_Lexica.git
   spacy
-  holmes_extractor
   coreferee
   rdflib
   spacytextblob
-  numpy
+  numpy==1.26.4
```

Comment:
I'm reluctant to add version locking here without knowing where the code breaks. Have we figured out why we need both this and the torch version lock?
Ideally, we don't have to lock either and we just fix the code that is breaking.